google/cloud/aiplatform/models.py

```diff
@@ -1174,7 +1210,16 @@ def _undeploy(
         self._sync_gca_resource_if_skipped()
         current_traffic_split = traffic_split or dict(self._gca_resource.traffic_split)
 
-        if deployed_model_id in current_traffic_split:
+        # TODO(b/211351292): Remove check once traffic split is supported
+        # Skip unallocating traffic and raise if new split is provided
+        if self.network:
+            if traffic_split:
+                raise ValueError(
```

Comment:
Preference to allow the service to raise in this case.

Reply:
Fixed, moved to the `undeploy()` method of private Endpoint